### PR TITLE
Fix export

### PR DESCRIPTION
**Pull request by senadglojnaric**
Hello, here is my Pull Request for fixing an issue #1070 . I also added a test in edge-cases.

### CODE REVIEW

**Review by dfahlander — COMMENTED** [review]
I don't see how this fixes the issue. The unit test has to reproduce the error. Does the unit test fail if you remove your fix?

File: addons/dexie-export-import/src/export.ts [hunk 1]
```diff
@@ -85,7 +85,13 @@ export async function exportDB(db: Dexie, options?: ExportOptions): Promise<Blob
       const table = db.table(tableName);
       const {primKey} = table.schema;
       const inbound = !!primKey.keyPath;
-      const LIMIT = options!.numRowsPerChunk || DEFAULT_ROWS_PER_CHUNK;
+
+      //issue #1070 //////////////////////////////////
+      const rowsPerChunk= options!.numRowsPerChunk || DEFAULT_ROWS_PER_CHUNK; 
+      const rowCount = await table.count(count => count);
```
Comment (dfahlander) [inline addons/dexie-export-import/src/export.ts:91]: Why is this needed?

**Comment by senadglojnaric** [comment]
Yes, the unit test fails if the fix is removed.

File: addons/dexie-export-import/src/export.ts (hunk 1 above)
Comment (ghost) [reply addons/dexie-export-import/src/export.ts:91]: Extra comma appears only when the numRowsPerChunk is smaller then the number of rows in the table, wether is defined by the user or a default value.

**Comment by dfahlander** [comment]
But you are not solving the problem. The reason for having the option rowsPerChunk is to save memory. The bug is not solved by working around the issue and increasing the limit. The whole idea is that we want to save memory here, so that databases with a million rows can be exported in chunks. The bug is that this fails if there's a filter function.

I close this for now.
